lib/scrape.js

```diff
@@ -48,6 +49,23 @@ try {
   console.log('No existing data...')
 }
 
+const FEED_LINKS = [
+  'http://systers-opensource.blogspot.com/feeds/posts/default',
+  'https://www.haiku-os.org/blog/index.xml',
+  'https://blog.wikimedia.org/feed/',
+  'http://www.liquidgalaxylab.com/feeds/posts/default',
+  'https://blog.zulip.org/rss/',
+  'https://blog.musicbrainz.org/feed/',
+  'https://insights.ubuntu.com/feed/',
+  'http://www.xwiki.org/xwiki/bin/view/Main/WebRss?xpage=rdf',
+  'https://www.drupal.org/planet/rss.xml',
+  'http://planet.osgeo.org/atom.xml',
+  // Not formateed XML 'https://blog.coala.io/atom.xml',
+  'http://mifos.org/feed/',
+  'https://blog.fossasia.org/feed/',
+  'http://openmrs.org/feed/',
```

Comment:
This has to come from `blog_url` not hardcoded.

Reply:
The reason I hardcoded is that coala's feed link has some problem(it is not formatted well) and it breaks the program.

Reply:
Then make the program not break :P

Reply:
OK:)
Can I ignore coala's feed for now?  I opened the xml on my machine, it is empty.

Reply:
Make it so that when the parser failed its skipping instead of "breaking".